awx/ui_next/src/components/LaunchButton/LaunchButton.jsx

```diff
@@ -146,6 +151,7 @@ function LaunchButton({ resource, i18n, children, history }) {
       }
     } catch (err) {
       setError(err);
+      setIsSending(false);
```

Comment:
```suggestion
    setError(err);
     } finally {
      setIsSending(false);
     }
```

Reply:
I thought about this too, but since we navigate away from the page on button click I wasn't sure it was necessary.  Probably good to have it just to be safe.